nf1_cellprofiler_classifier_fs_all_cellprofiler_all_feat_perfect.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>Just in my opinion, I feel like it would be nice to see a few code comments explaining what is going on (especially in the second line). It is good for the future if you choose to change anything like the <code>train_size</code>.</p><p> </p><p>But, you don't have to add this, just my suggestion! </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/NF1_SchwannCell_data_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
True, there can never be enough documentation!